editor/src/components/inspector/common/inspector-end-to-end-tests.spec.browser.tsx

```diff
@@ -519,10 +519,10 @@ describe('inspector tests with real metadata', () => {
       'opacity-number-control',
     )) as HTMLInputElement
     const minWidthControl = (await renderResult.renderedDOM.findByTestId(
-      'position-Width-number-input',
```

Comment:
@enidemi I found that the endtoend tests were using the wrong controls for minWidth and maxWidth, I fixed them here